src/main/java/frc/robot/commands/AcquireAllSidesCommand.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Acquisition;
+
+public class AcquireAllSidesCommand extends Command {
+    private static Acquisition acquisition = Acquisition.getInstance();
+
+    public AcquireAllSidesCommand() {
+        addRequirements(acquisition);
+    }
+
+    @Override
+    public void execute() {
+        acquisition.runFront();
```

Comment:
Didn't we set these motors as a follow? If so please rename runFront to acquire and make sure there is a dispose function as well